Client/Frontend/Browser/Onboarding/OnboardingAdsCountdownViewController.swift

```diff
@@ -44,8 +44,10 @@ class OnboardingAdsCountdownViewController: OnboardingViewController, UNUserNoti
         contentView.animate(from: 0.0, to: 1.0, duration: UX.animationTime) { [weak self] in
             guard let self = self else { return }
             
-            //Show the confirmation screen no matter what..
-            self.contentView.setState(.finished)
+            //Show the confirmation screen no matter what.. after 1 second delay
+            DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
+                self.contentView.setState(.finished)
```

Comment:
What happens if the user taps/dismisses the ad before this executes?

Reply:
That's exactly the problem with doing this.. but it's a ticket I have to fix and official spec. 

Honestly, I have no idea what "should" happen. What does happen though, is that it goes straight to the browser and marks onboarding as complete even if they never saw the ad because they decided to tap it really fast.

Likewise what happens when they tap both (as per previous comment)?

Reply:
@kylehickinson @Brandon-T 
If you tap "Start Browsing", it should go straight to the browser.
If you tap the ad, it should go straight to the browser with a new tab opened to www.brave.com/my-first-ad/

It is not possible to "tap both at the same time" because either action will remove the possibility of performing the other action.